stonesoup/platform/tests/test_platform_base.py

```diff
@@ -534,19 +536,32 @@ def test_multi_transition():
 
     platform = MultiTransitionMovingPlatform(transition_models=transition_models,
                                              transition_times=transition_times,
-                                             state=platform_state,
+                                             states=platform_state,
                                              position_mapping=[0, 2],
                                              sensors=None)
 
     assert len(platform.transition_models) == 2
     assert len(platform.transition_times) == 2
 
-    px, py = platform.position[0], platform.position[1]
+    #  Check that platform states lenght increases as platform moves
+    assert len(platform) == 1
     time = datetime.datetime.now()
+    time += datetime.timedelta(seconds=1)
+    platform.move(timestamp=time)
+    assert len(platform) == 2
+    time += datetime.timedelta(seconds=1)
+    platform.move(timestamp=time)
+    assert len(platform) == 3
+    time += datetime.timedelta(seconds=1)
+    platform.move(timestamp=time)
+    assert len(platform) == 4
```

Comment:
For the sake of having less lines of code, is it worth putting these checks in a FOR loop?